packages/channels/src/messenger/README.md

```diff
@@ -1,29 +1,31 @@
+# Messenger (v1.0.0)
+
 ### Sending
 
-| Channels | Messenger |
-| -------- | :-------: |
-| Text     |    ✅     |
-| Image    |    ✅     |
-| Choice   |    ✅     |
-| Dropdown |    ✅     |
-| Card     |    ✅     |
-| Carousel |    ✅     |
-| File     |    ✅     |
-| Audio    |    ✅     |
-| Video    |    ✅     |
-| Location |    ✅     |
+| Channels | Messenger | Details                    |
+| -------- | :-------: | :------------------------- |
+| Text     |    ✅     |                            |
+| Image    |    ✅     |                            |
+| Choice   |    ✅     |                            |
+| Dropdown |    ✅     |                            |
+| Card     |    ✅     |                            |
+| Carousel |    ✅     |                            |
+| File     |    ✅     | File sent as text link     |
```

Comment:
Why not `sent as url`? The more wordy and correct way to say it would be `text for title with a link to the file`. If we want to be more concise it's fine but why use `text link` as a sort of roundabout way of saying that there's a text part and a link part.